airbyte-ci/connectors/metadata_service/lib/tests/fixtures/__init__.py

```diff
@@ -3,15 +3,26 @@
 from typing import List
 
 
+def list_all_paths_in_fixture_directory(folder_name: str) -> List[str]:
+    file_path = os.path.join(os.path.dirname(__file__), folder_name)
+    return [os.path.join(file_path, file_name) for file_name in os.listdir(file_path)]
+
+
 @pytest.fixture
 def valid_metadata_yaml_files() -> List[str]:
-    file_path = os.path.join(os.path.dirname(__file__), "valid")
-    # list the absolute paths of all files in the directory
-    return [os.path.join(file_path, file_name) for file_name in os.listdir(file_path)]
+    return list_all_paths_in_fixture_directory("metadata_validate/valid")
 
 
 @pytest.fixture
 def invalid_metadata_yaml_files() -> List[str]:
-    file_path = os.path.join(os.path.dirname(__file__), "invalid")
-    # list the absolute paths of all files in the directory
-    return [os.path.join(file_path, file_name) for file_name in os.listdir(file_path)]
+    return list_all_paths_in_fixture_directory("metadata_validate/invalid")
+
+
+@pytest.fixture
```

Comment:
nit: as these files are static I think its worth using the session scope: the fixtures will only be called once per test session and their results will be re-used throughout tests.
```suggestion
@pytest.fixture(scope="session")
```